_rules/focus-returns-trigger-9au0ou.md

```diff
@@ -0,0 +1,188 @@
+---
+id: 9au0ou
+name: Focus returns to trigger
+rule_type: atomic
+description: |
+  This rule checks that when a modal closes the focus returns to the trigger that opened the modal.
+accessibility_requirements:
+  wcag20:2.4.3: # Focus Order (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G59: # Placing the interactive elements in an order that follows sequences and relationships within the content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - DOM Tree
+acknowledgments:
+  authors:
+    -  Helen Burge
+  previous_authors:
+---
+
+## Applicability
+
+This rule applies to any [modal][] or item with the role of `dialog` + `aria-modal="true"` that when no longer visible, the [focus][] returns to the [instrument][] or [trigger][] that made the [modal][] visible.
```

Comment:
Some issues I see with this:
* This seems to mix the Applicability and intention of the rule. Especially, the "when no longer visible, the focus returns to…" part is what the rule actually checks, not what it applies to.
* I'm not sure we can really use "modal" from the ARIA authoring practices as a definition here. OTOH, we should be good by looking at `dialog` elements + `role="modal"` ones.
* By targeting an already opened modal, we do not really know what was the instrument used to open it. I.e. if there are two buttons to open the modal, but the test starts when the modal is already opened, how can we check that we return to the correct one. I think the Applicability should either target the instrument to open a modal, or the combination of the open modal plus the instrument that opened it. Targeting the instrument seems easier (i.e. it means that opening the modal is part of the test).
* I'm not sure anymore where we are with the full "test instructions" bit (i.e. this rule requires some precise manipulations). I think this is part of the "stateful rules" discussion. Let's just assume the problem will be solved elsewhere and not care about it here 😄 We may have to adapt the rule according to the decision reached, however.
* We can probably use the "open/close" modal vocabulary only (i.e.  not mention visibility, or enter/exit). This is linked to either `aria-modal` or the `showModal` status in HTML sepcs, so we have both the technical solid ground fnor it, and easy enough vocabulary to understand.